tests/tests_C2_economic_functions.py

```diff
@@ -32,54 +35,66 @@ def test_crf():
     assert CRF == (0.1 * (1 + 0.1) ** 20) / ((1 + 0.1) ** 20 - 1)
 
 
-def test_capex_from_investment(lifetime):
+def test_capex_from_investment_lifetime_equals_project_life():
     """
 
-    Tests whether the MVS is correctly calculating the capital expenditure of the project
+    Tests whether the MVS is correctly calculating the capital expenditure of the project if the lifetime is equal to project_life
     """
-    for years in lifetime:
-        if years == project_life:
-            CAPEX = e_functions.capex_from_investment(investment_t0, years, project_life,wacc,tax)
-            assert CAPEX == 220000*(1+0.15)
-        else:
-            CAPEX = e_functions.capex_from_investment(investment_t0, years, project_life, wacc, tax)
-            first_investment = 220000*(1+0.15)
-            assert CAPEX == first_investment + first_investment / (1 + 0.1) ** (15 * 1) + first_investment \
-                   + first_investment / (1 + 0.1) ** (15 * 1) + first_investment / (1 + 0.1) ** (15 * 2) \
-                   - ((first_investment / ((1 + 0.1) ** 15)) / 15) * (2 * 15 - 20)
+    CAPEX = e_functions.capex_from_investment(investment_t0, lifetime[0], project_life, wacc, tax)
+    assert CAPEX == 220000 * (1 + 0.15)
```

Comment:
```suggestion
    assert CAPEX == investment_t0 * (1 + 0.15)
```
Also, what is  (1 + 0.15)?

Reply:
1 + 0.15 = 1 + tax and tax is defined at the start